ec_grasp_planner/src/planner.py

```diff
@@ -200,26 +204,57 @@ def handle_run_grasp_planner(self, req):
 
         return plan_srv.RunGraspPlannerResponse(ha.xml())
 
+
```

Comment:
I think this is not useful, there is always a `general object` param set, when you instantiate a new type of object then then you copy all the param set from the `general object`. So, there should not be any param in object that is not present in the general object!

Reply:
I agree on your statement, but I don't quite get to which part of the code your comment belongs. It only points to an added newline (line 207)